plugins/faustwp/includes/settings/callbacks.php

```diff
@@ -239,6 +239,14 @@ function sanitize_faustwp_settings( $settings, $option ) {
 				}
 				break;
 
+			case 'telemetry_client_id':
+				if ( $value ) {
+					$settings[ $name ] = sanitize_text_field( $value );
+				} else {
+					unset( $settings[ $name ] );
+				}
+				break;
+
```

Comment:
This is a duplicate of the `enable_telemetry` case. But, it's very important we don't mess that one up, so I kept them separate in case we update one or the other.